macros/src/main/scala/clue/macros/GraphQLMacro.scala

```diff
@@ -59,12 +61,17 @@ protected[macros] trait GraphQLMacro extends Macro {
     def fromGrackleType(
       name:         String,
       tpe:          Type,
+      isInput:      Boolean,
       mappings:     Map[String, String],
       nameOverride: Option[String] = None
     ): ClassParam = {
       def resolveType(tpe: Type): Tree =
         tpe match {
-          case NullableType(tpe) => tq"Option[${resolveType(tpe)}]"
+          case NullableType(tpe) =>
+            if (isInput)
+              tq"clue.data.Input[${resolveType(tpe)}]"
```

Comment:
`_root_.clue`?

Reply:
This is just to pretend I know how to do macros

Reply:
Good catch.